pegasus/sites.v3/code.org/public/starwars.haml

```diff
@@ -55,3 +55,18 @@ social:
 %div.starwars-videos{style: "clear: both;"}
   %h2.sw-header2= I18n.t(:starwars_watch_videos)
   =view :inspirational_videos
+
+= view 'mobilecheck.js'
+
+:javascript
+
+  $(document).ready(function() {
+    window.isIOS = function() {
+      var reg = /iP(hone|od|ad)/;
+      return reg.test(window.navigator.userAgent);
+    };
+
+    if (! window.isIOS() && ! window.mobilecheck()) {
```

Comment:
Do we have a styleguide for js? I usually would do `if (!window.isIOS() && !window.mobilecheck())`